docs/intents/ref/StartChat.md

```diff
@@ -53,10 +59,23 @@ const initSettings = {
         groupRecipients: true, // one chat with both contacts
         public: false, // private chat room
         allowHistoryBrowsing: true,
-        allowMessageCopy: true,
-        allowAddUser: false, // John won't be authorized to add other users to the chat
+        allowMessageCopy: true
+    }
+    message: {
+      type: 'fdc3.message',
+      text: {
+        'text/plain': 'Hey all, can we discuss the issue together? I attached a screenshot'
+      },
+      entities: {
+         '0': {
+             type: 'fdc3.fileAttachment',
+              data: {
```

Comment:
data element to be removed

Reply:
i think this one is OK? i've remove all those on Action objects